chatbot.py

```diff
@@ -8,30 +8,32 @@
 from utilities import learn_question_response
 from utilities import add_learnt_statement_to_database
 from utilities import setup_nltk
-from googleMapsApiModule import direction
-from googleMapsApiModule import add_to_maps_database
-from googleMapsApiModule import get_from_maps_database
-from googleMapsApiModule import geocoding
-
+import enum
+class symbolic_constant(enum.Enum): 
+    msg=0
+    train_me=1
+    ogn=2
+    desti=3
```

Comment:
Best practise - class names are usually CamelCase rather than snake_case. Also, more descriptive names would be better than truncated ones for enum keys. Have a look at `auto()` provided by enum.

```python3
class LearnResponse(enum.Enum): 
    message = auto()
    train_me = auto()
    origin = auto()
    destination = auto() 
```